libs/ui-toolkit/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -73,7 +73,7 @@ export const DropdownMenuContent = forwardRef<
   <DropdownMenuPrimitive.Content
     {...contentProps}
     ref={forwardedRef}
-    className="min-w-[290px] bg-neutral-200 dark:bg-white p-2 rounded"
+    className="min-w-[290px] bg-neutral-200 dark:bg-white p-2 rounded text-black"
```

Comment:
fixes this issue 
<img width="1002" alt="Screenshot 2023-02-28 at 14 53 29" src="https://user-images.githubusercontent.com/16125548/221891371-b8e5a971-d9ac-4c1c-803b-30ead6b7c41e.png">